test/ExportVerilog/sv-dialect.mlir

```diff
@@ -240,3 +240,14 @@ rtl.module @issue439(%in1: i1, %in2: i1) {
     sv.fwrite "Bye %x\n"(%merged) : i1
   }
 }
+
+rtl.module.extern @Foo() -> ()
+
+// CHECK-LABEL: module issue625(
+rtl.module @issue625() -> (%x: i1) {
+  %1 = comb.constant(1 : i1) : i1
+  %o = comb.add %1, %1 {name="Foo"} : i1
```

Comment:
We allow a name attribute on comb operations?  This is really dangerous, where did this get added?

Reply:
Why is it dangerous? All name "discardable" attributes go through the name uniquer, so they're not guarantees so much as suggestions. This is very helpful (essential even) to produce readable SystemVerilog. ESI uses it extensively to create very readable SystemVerilog.

Reply:
They're dangerous in that people will come to expect them to work, but they cannot - because canonicalization and other things will drop them.

I'm not actually even sure how this works - this would look like an 'inherent' attribute, but isn't declared by the operation.

I'm ok with this be a non-inherent attribute I guess, but I'm just confused how this is "working" right now.